instagram_app/src/Components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './CommentSection.css';
+
+
+function CommentSection (props) {
+    return(
+<div className="container">
+<div className="userHeader">
+    <img src={props.eachPost.thumbnailUrl} alt="thumbnail" />
+    <h2>{props.eachPost.username}</h2>
+</div>
+<img src={props.eachPost.imageUrl} alt="imageurl" className='postImage'/>
+<div className="likesComments">
+    <img src="https://img.icons8.com/ios/30/000000/like.png"></img>
+    <img src="https://img.icons8.com/ios/30/000000/topic.png"></img>
+</div>
+<div className='counter'>
+    <p className="likes">{props.eachPost.likes} likes</p>
+</div>
+
+
+
+<input type="text" placeHolder="Add a comment..." onChange={props.inputHandler} value={props.input}></input>
```

Comment:
I am missing your inputHandler logic in your project. I would wrap your input in a form to make it easier for you to submit the input that a user types in. This input needs to be held on state for a while somewhere in your application until it is pushed to the comments array for that user.

Reply:
If you need help with this, please let me know and we can go through it!